hack/lib/cleanup.sh

```diff
@@ -94,8 +94,8 @@ function os::cleanup::tmpdir() {
         done
     fi
 
-		# delete any sub directory underneath BASETMPDIR
-    for directory in $( find "${BASETMPDIR}" -d 2 ); do
+    # delete any sub directory underneath BASETMPDIR
+    for directory in $( find "${BASETMPDIR}" -maxdepth 2 ); do
```

Comment:
The `find` doc says:

```
find: warning: the -d option is deprecated; please use -depth instead, because the latter is a POSIX-compliant feature.
```